### PR TITLE
fix: update axios to 1.6.1 to fix CVE-2023-45857

### PR DESCRIPTION
**Pull request by AndrewMax**
Axios was recently updated to fix CVE-2023-45857. This PR is to update to axios 1.6.0.

### CODE REVIEW

**Comment by seanputera** [comment]
Thank you!

**Comment by AndrewMax** [comment]
@jeffbski Can this get some attention please? Thanks a lot.

**Comment by pat-s** [comment]
And update and a subsequent release would be great!

**Review by mboere-seek — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "temp": "^0.9.4"
   },
   "dependencies": {
-    "axios": "^0.27.2",
+    "axios": "^1.6.0",
```
[the remark this answers is not in the document]
Comment (AndrewMax) [reply package.json]: @porschesstein Thanks, updated. When I originally opened this PR, the latest version of Axios was still 1.6.0.

**Review by seanputera — APPROVED** [review]

**Comment by benasher44** [comment]
Guess we'll just have to wait-on this PR.

**Comment by wellwelwel** [comment]
Tested locally and worked perfectly 🚀

**Review by wellwelwel — APPROVED** [review]

**Comment by littleamigo** [comment]
I hope this gets merged soon and then released! Looking forward...

**Comment by jeffbski** [comment]
Thanks @AndrewMax for the PR and for those that confirmed it. It is published to wait-on@7.2.0 

https://github.com/jeffbski/wait-on/releases/tag/v7.2.0

**Pull request merged**